packages/ng/forms/multilanguage-input/multilanguage-input.component.scss

```diff
@@ -0,0 +1,14 @@
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+.popover-contentOptional {
+
+	padding: var(--pr-t-spacings-100) !important;
+	max-height: 20rem;
```

Comment:
Can we imagine `max-height` being a variable?

Reply:
If you see use cases, yep!